src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -531,6 +531,9 @@ void VtolAttitudeControl::task_main()
 			// update transition state if got any new data
 			if (got_new_data) {
 				_vtol_type->update_transition_state();
+
+				_vtol_type->process_mc_data();
+				fill_mc_att_rates_sp();
```

Comment:
Oh yes stupid me. My proposition doesn't make any sense. But then again I'm confused about what this does anyway...